internal/apiserver/route_spi_get_ops.go

```diff
@@ -37,6 +37,7 @@ var spiGetOps = &ffapi.Route{
 	Extensions: &coreExtensions{
 		FilterFactory: database.OperationQueryFactory,
 		CoreJSONHandler: func(r *ffapi.APIRequest, cr *coreRequest) (output interface{}, err error) {
+			// TODO: cr.or will be nil (this must be converted to a namespaced query)
```

Comment:
Should we not do this in this PR? Having a broken endpoint seems bad, and it should be fine to make it `operations/{ns}`

Reply:
(We still need the `operations/{nsopid}` routes for `GET` and `PATCH`, where the namespace is embedded into the ID)

Reply:
Yep that works - I can replace that route. Just wanted to get your feedback on how to spell it in the context of the other SPI routes and the usage by FFTM.